clevis/Sanity/kernel-keyring-support/testinfo.desc

```diff
@@ -0,0 +1,15 @@
+Owner:           Martin Zeleny <mzeleny@redhat.com>
+Name:            /CoreOS/clevis/Sanity/kernel-keyring-support
+TestVersion:     1.0
+Path:            /mnt/tests/CoreOS/clevis/Sanity/kernel-keyring-support
+Description:     Test clevis to provide a parameter for the key description of a key in the kernel keyring
```

Comment:
Same comment as before regarding description

Reply:
This file won't go to the repo.